### PR TITLE
Show user roles id 58103

### PR DESCRIPTION
**Pull request by josejtax**
Trac ticket: 

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/users.php [hunk 1]
```diff
@@ -370,13 +370,17 @@
...
-				wp_dropdown_users(
-					array(
-						'name'    => 'reassign_user',
-						'exclude' => $userids,
-						'show'    => 'display_name_with_login',
-					)
-				);
+				$users = get_users( array( 'exclude' => $userids ) );
+					usort($users, function($a, $b) {
+						$role_a = $a->roles[0];
```
Comment (lphoumpakka) [inline src/wp-admin/users.php:375]: Check empty role
Prefer use reset

File: src/wp-admin/users.php [hunk 2]
```diff
@@ -370,13 +370,17 @@
...
+						$role_b = $b->roles[0];
+						return strcmp($role_a, $role_b);
+					});
+					echo '<select name="reassign_user">';
+					foreach ( $users as $user ) {
+						echo '<option value="' . esc_attr( $user->ID ) . '"> User: ' . esc_html( $user->display_name ) . ' - rol: ' . esc_html( $user->roles[0] ) . '</option>';
```
Comment (lphoumpakka) [inline src/wp-admin/users.php:381]: Missing "e" to "role".
Needs translation for "User" and "role".
Add more check to empty role 
Prefers use reset for $user->roles

**Review by lphoumpakka — CHANGES_REQUESTED** [review]

**Comment by hz-tyfoon** [comment]
Looks like there are some `coding standards issues` in the file changes. Can u fix them?

**Comment by audrasjb** [comment]
Closed in favor of #4441

**Pull request closed without merging**